packages/client/src/components/SafeAssets.js

```diff
@@ -67,7 +67,6 @@ function SafeAssets({ web3, name, address }) {
             onClick={() =>
               modalContext.openModal(
                 <SendTokens
-                  name={name}
```

Comment:
removed the name property since that's not being used in the new design of sendToken flow